src/test/container-features/configs/dockerfile-with-v2-oci-features/.devcontainer.json

```diff
@@ -7,7 +7,7 @@
 	},
 	"features": {
 		"terraform": "latest",
-		"ghcr.io/devcontainers/features/docker-in-docker@sha256:e32e8937c87345ff7a937d22cacb7f395d41deffde9943291ef3cc0ac91a8ac6": {},
+		"ghcr.io/devcontainers/features/docker-in-docker@sha256:440bdb81cf8af43f3e922450d33db9775c1097340557a5b7b6fe705bc758c5ef": {},
```

Comment:
I agree this seems like the best fix.  (I did the same thing in this code that didn't end up merging: https://github.com/devcontainers/cli/pull/771/files#diff-8ce1ff732a81b4f6c20dff0d3b7bff5d76d973c63b2fc5640d741fd8aca08770)